src/main/java/uk/gov/companieshouse/api/strikeoffobjections/config/ValidationConfig.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.companieshouse.api.strikeoffobjections.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.companieshouse.api.strikeoffobjections.common.ApiLogger;
+import uk.gov.companieshouse.api.strikeoffobjections.model.entity.ObjectionStatus;
+import uk.gov.companieshouse.api.strikeoffobjections.validation.AllowedValuesValidationRule;
+import uk.gov.companieshouse.api.strikeoffobjections.validation.DisallowedValuesValidationRule;
+import uk.gov.companieshouse.api.strikeoffobjections.validation.ValidationRule;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Configuration
+public class ValidationConfig {
+
+    @Value("#{'${ACTION_CODES_COMPANY_STRUCK_OFF}'.split(',')}")
```

Comment:
Nice. Didn't know you could inline the split() call.

Reply:
good ole google, it even converts them into Longs